docs/design/authentication-patterns.md

```diff
@@ -6,20 +6,15 @@ Add-ins may require users to sign-in or sign-up in order to access features and
 
 |Do|Don't|
 |:----|:----|
-|Use single sign-on (SSO) to authenticate users into your add-in.|Require users to sign into your add-in separately from their personal Microsoft account or their Office 365 account (work or school).|
 |Prior to sign-in, describe the value of your add-in or demonstrate functionality without requiring an account.	|Expect users to sign-in without understanding the value and benefits of your add-in.|
 |Guide users through authentication flows with a primary, highly visible button on each screen.	|Draw attention to secondary and tertiary tasks with competing buttons and calls to action.|
 |Use clear button labels that describe specific tasks like "Sign in" or "Create account".	|Use vague button labels like "Submit" or "Get started" to guide users through authentication flows.|
 |Use a dialog to focus users' attention on authentication forms.	|Overcrowd your task pane with a first run experience and authentication forms.|
 |Find small efficiencies in the flow like auto-focusing on input boxes.	|Add unnecessary steps to the interaction like requiring users to click into form fields.|
 |Provide a way for users to sign-out and reauthenticate.	|Force users to uninstall to switch identities.|
 
-> [!NOTE]
-> The Single Sign-on API is currently supported in preview for Word, Excel, Outlook, and PowerPoint. For more information about where the Single Sign-on API is currently supported, see [IdentityAPI requirement sets](https://docs.microsoft.com/office/dev/add-ins/reference/requirement-sets/identity-api-requirement-sets?view=office-js). If you are working with an Outlook add-in, be sure to enable Modern Authentication for the Office 365 tenancy. For information about how to do this, see [Exchange Online: How to enable your tenant for modern authentication](https://social.technet.microsoft.com/wiki/contents/articles/32711.exchange-online-how-to-enable-your-tenant-for-modern-authentication.aspx).
-
-
 ## Authentication flow
-If single sign-on is not yet available to your users, consider an alternative authentication flow. Give users a choice to sign-in directly with your service or an identity provider like Microsoft.
+Until single sign-on is out of preview, production add-ins should give users a choice to sign-in directly with your service or an identity provider like Microsoft.
```

Comment:
Consider changing "out of preview" to "publicly available". That way, readers don't have to understand our release cadence or terminology.

Reply:
I was going to do this, but the more I think about it, the more I think that "in preview" and "out of preview" are more meaningful and clear to our developer audience than "publicly available" which is kind of ambiguous. A preview is not "private" or unavailable (to the dev audience).